security/email-templates.md

```diff
@@ -0,0 +1,131 @@
+# Envoy Security Process Email Templates
+
+This is a collection of email templates to handle various situations the security team encounters.
+
+## Upcoming security release to envoy-announce@googlegroups.com
+
+```
+Subject: Upcoming security release of Envoy $VERSION
+To: envoy-announce@googlegroups.com
+Cc: envoy-security@googlegroups.com, envoy-maintainers@googlegroups.com
+
+Hello Envoy Community,
+
+The Envoy maintainers would like to announce the forthcoming release of Envoy
+$VERSION.
+
+This release will be made available on the $ORDINALDAY of $MONTH $YEAR at
+$PDTHOUR PDT ($GMTHOUR GMT). This release will fix $NUMDEFECTS security
+defect(s). The highest rated security defect is considered $SEVERITY severity.
+
+No further details or patches will be made available in advance of the release.
```

Comment:
Is there any reason why CVEs are not listed in the public announcement?

Reply:
I didn't see it in the OpenSSL security release announcements. Is this something that is done?

Reply:
It seems that they don't do that, indeed. I was mostly curious about the reasoning for including them in private, but not public announcements. I don't have any opinions one way or the other.

Reply:
In private comms we're also disclosing the description of the CVE and candidate patches, which need to be correlated and tracked with the public disclosure. We maintain radio silence with details on the public announcement until the fix release. I'm open to hearing about other practices, this is just a strawman.

Reply:
Sounds reasonable, thanks.